### ISSUE TITLE
Yolov4-Tiny EdgeTPU Missing Leaky Relu

### ISSUE THREAD

**Issue by benlemond** [PINTO0309/PINTO_model_zoo#51]
Hello,
How were you able to execute a Yolo on the Coral with Leaky Relu?  We are using tfLite interpreter with "libedgetpu.so.1.0" and we are getting errors.
`self.interpreter = tf.lite.Interpreter(
                hw_model_path,
                experimental_delegates=[load_delegate('libedgetpu.so.1.0')]
            )
`
`main.ERROR - Only float32 and uint8 is supported currently, got INT8.Node number 404 (LEAKY_RELU) failed to invoke.`

**Comment by PINTO0309** [comment]
@benlemond 
I tried to generate the EdgeTPU model by replacing LeakyReLU with PReLU. Try it once.
**https://github.com/PINTO0309/PINTO_model_zoo/tree/master/046_yolov4-tiny/20_Replace_LeakyReLU_to_PReLU**

**Comment by benlemond** [comment]
@PINTO0309 
`main.ERROR - Only float32 and uint8 are supported currently, got -1695564548.Node number 5 (PRELU) failed to invoke.`

**Comment by PINTO0309** [comment]
@benlemond 

I found the source of the problem.
**https://github.com/google-coral/edgetpu/issues/187**

It seems to be a bug when the **`TFLite converter`** converts **`ResizeNearestNeighbor`**.
**`half_pixel_centers`** should not be `True`, but will be forced to be converted to `True`.

I think it's a matter of the converter, so it's out of my hands at this point.

**Comment by PINTO0309** [comment]
@benlemond 
Try it if you have the time.
- EdgeTPU, YOLOv4-tiny, INT8
**https://drive.google.com/file/d/1NsJlF-MVxBw7JCw60K1uFLTAlwdv-C0p/view?usp=sharing**

![Screenshot 2020-11-26 22:11:51](https://user-images.githubusercontent.com/33194443/100355551-51e08280-3035-11eb-9987-0fb7cbb4bbb3.png)

**Comment by gouliang1997** [comment]
I gave it a try and succeeded in getting these five outputs. But I don't know what to do with the data later.
 I think I should get two outputs, one for detected_boxes and one for scores.

**Comment by PINTO0309** [comment]
I have converted the model in the following repositories, so please refer to the following for post-processing.
https://github.com/bubbliiiing/yolov4-tiny-keras.git

**Comment by gouliang1997** [comment]
I processed the output according to the process in the repo, but the x y w h value of the output of this model is between 40 and 60.(output_details[3][..., :4] and output_details[4][..., :4]) 
If I put these values into the sigmoid function, I will get the output all turn out to be 1 because these values are obviously too large.
I think I need to make some adjustments to the output of the model but I don't know how
